coralproject/talk
# [CORL-2178] Fix missing toxicity score on moderate cards

## What does this PR do?

Makes sure the automated actions tab visibility isn't only bound to their being external mod phases, but also toxicity scores if available. This bug happened because we moved the perspective toxicity scores under the automated actions tab which was originally built to only show external mod phase results.

Now the tab visibility is keyed to both external mod phases and toxicity data.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Hook up perspective in config
- Create some comments
- Look at them in the mod queues
- Check the mod card's Details > Automated Actions
- See the toxicity score
